CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required (VERSION 2.8)
 project(OgreVideo)
 
-find_package(OGRE 1.11 REQUIRED COMPONENTS Bites)
+set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake")
```

Comment:
do not modify the module path. Instead put the FindOGRE.cmake for 2.1 in one of the predefined search locations:
https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure